addons/website_forum/views/forum_forum_templates_post.xml

```diff
@@ -506,7 +506,7 @@
 
 <!-- Utility template: Post a Comment -->
 <template id="post_comment">
-    <div class="o_wforum_post_comments_container rounded">
+    <div class="o_wforum_post_comments_container rounded d-flex flex-column gap-2">
```

Comment:
Hello CHGO, nice catch, it is indeed better when we separate de comments and I guess this is how it is supposed to be.

Reply:
> Also note that in the structure of the comment list it seems like either o_wforum_post_comments or o_wforum_post_comment is unnecessary but I think it's better to wait for @edi-odoo's 

Hey 👋
After some investigating I've come to the conclusion that the flex styles on `o_wforum_post_comments` used to be used for the same purpose but something changed along the way and they got left there. So you can indeed remove them.

Thanks for your fix!